tests/accessor/accessor_default_values_fp64.cpp

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for accessor default values.
+//
+//  This test provides verifications that template parameters has default values
+//  for generic accessor, host_accessor and local_accessor with double type.
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+#include "../common/disabled_for_test_case.h"
+
+// FIXME: re-enable when sycl::accessor is implemented
+#if !defined(__HIPSYCL__) && !defined(__COMPUTECPP__) && \
+    !defined(__SYCL_COMPILER_VERSION)
+
+#include "accessor_default_values.h"
+
+using namespace accessor_default_values_test;
+using namespace accessor_tests_common;
+#endif
+
+namespace accessor_default_values_test_fp64 {
+using namespace sycl_cts;
+
+DISABLED_FOR_TEST_CASE(hipSYCL, ComputeCpp, DPCPP)
+("Accessors constructor default values test fp64 types.", "[accessor]")({
+  auto queue = sycl_cts::util::get_cts_object::queue();
+  if (!queue.get_device().has(sycl::aspect::fp64)) {
+    WARN(
+        "Device does not support double precision floating point operations. "
+        "Skipping the test case.");
+    return;
+  }
+
+#ifdef SYCL_CTS_ENABLE_FULL_CONFORMANCE
```

Comment:
```suggestion
#if SYCL_CTS_ENABLE_FULL_CONFORMANCE
```

Reply:
I've applied this comment in this commit: https://github.com/KhronosGroup/SYCL-CTS/pull/335/commits/66c0562df41dca778c91018c6669d171f18b0807